cpt/packager.py

```diff
@@ -143,6 +144,8 @@ def __init__(self, username=None, channel=None, runner=None,
         else:
             self.upload_only_when_tag = get_bool_from_env("CONAN_UPLOAD_ONLY_WHEN_TAG")
 
+        self.upload_only_recipe = upload_only_when_tag or get_bool_from_env("CONAN_UPLOAD_ONLY_RECIPE")
```

Comment:
```suggestion
        self.upload_only_recipe = upload_only_recipe or get_bool_from_env("CONAN_UPLOAD_ONLY_RECIPE")
```
I don't think passing this argument is tested. The environment variable is tested though.

Reply:
Thanks for finding this bug. I'll add a test case to check this.